### DISCUSSION TITLE
Zoom off on AR side

### DISCUSSION THREAD

**Discussion by batuhanzorbeyzengin** [google/model-viewer#3385] [Q&A]
Hello,

I have such a model. When I switch to the AR side, my model opens at 100% scale, no problem. I want to turn off the zoom feature so that the model remains at its original size on the ar side. How can I do that. I couldn't find a resource on this.

https://test.digivegademo.site/kocelv1/

**Comment by elalish** [comment]
https://modelviewer.dev/docs/#entrydocs-augmentedreality-attributes-arScale

**Reply by batuhanzorbeyzengin** [reply]
ar-scale="fixed" I'm already using this tag if you examine the model in the link I gave you. But when you switch to the ar side, it can still be played on the model scale.

**Reply by elalish** [reply]
Which device and which of the three AR modes are you using? If you're not sure, please set `ar-mode` to one at a time so you can be sure. If you're on iOS, are you using an `ios-src` or auto-generation? There's a known issue with iOS, auto-generation, and scale=fixed (really their whole API).
